src/commands/Management/Message Filters/messageMode.ts

```diff
@@ -0,0 +1,24 @@
+import { CommandStore } from 'klasa';
+import { SelfModerationCommand } from '../../../lib/structures/SelfModerationCommand';
+import { GuildSecurity } from '../../../lib/util/Security/GuildSecurity';
+import { GuildSettings } from '../../../lib/types/settings/GuildSettings';
+
+export default class extends SelfModerationCommand {
+
+	protected $adder: keyof GuildSecurity['adders'] = 'messages';
```

Comment:
Why the $ sign 🤨

Reply:
It's a protected abstract field from SelfModerationCommand.
And because it's a keyof, the dollar kind of tells that it's an "accessor".

Reply:
fair enough